src/runtime_src/tools/scripts/pkgapu.sh

```diff
@@ -154,8 +156,8 @@ if [[ ! -d $IMAGES_DIR ]]; then
 fi
 IMAGES_DIR=`realpath $IMAGES_DIR`
 
-if [ -f $SETTINGS_FILE ]; then
-	source $SETTINGS_FILE
+if [[ ! (`which mkimage` && `which bootgen` && `which xclbinutil`) ]]; then
+	error "Please source Xilinx VITIS and Petalinux tools to make sure mkimage, bootgen and xclbinutil is accessible."
```

Comment:
@houlz0507 sourcing petalinux will not set mkimage path in PATH variable, so mkimage will not be picked automatically. That is the reason why I have hardcoded mkimage path earlier.

so users of this script can either install mkimage using **yum install -y uboot-tools** in centos/rhel machine to get mkimage binary or add petalinux mkimage path to PATH variable to autopick mkimage.

should we add setting of PATH variable in error message or it is okay?

Reply:
The mkimage path is shown in error message
Usage:  [options]
  options:
          -help                           Print this usage
          -images                         Versal images path
          -clean                          Remove build files
          -output                         output path
This script requires tools: mkimage, xclbinutil, bootgen, rpmbuild, dpkg-deb. 
There is mkimage in petalinux build, e.g.
/proj/petalinux/2021.2/petalinux-v2021.2_daily_latest/tool/petalinux-v2021.2-final/components/yocto/buildtools/sysroots/x86_64-petalinux-linux/usr/bin/mkimage

Reply:
looks good, I will update the BAI with this change. Thanks.